block-explorer/v3.yml

```diff
@@ -0,0 +1,643 @@
+          type: string
+        errors:
+          type: array
+          items:
+            type: string
+    Snapshot:
+      type: object
+      properties:
+        hash:
+          type: string
+        ordinal:
+          type: integer
+          format: int64
+        height:
+          type: integer
+          format: int64
+        subHeight:
+          type: integer
+          format: int64
+        lastSnapshotHash:
+          type: string
+        blocks:
+          type: array
+          items:
+            type: string
+        timestamp:
+          type: string
+          title: ISOString
+      example:
+        hash: 1ae6e0f9161b7a29c96e8accb853fb479e0ea026832850ff5def773b5ff00c00
+        ordinal: 58183
+        height: 58100
+        subHeight: 2
+        blocks:
+          - 0922bd4a63353040589547cfed00f2660fb736ce694199384f1df2adaac54fe6
+          - 0c27a295e1f58e41582538b8deb18a3aea4ae4b52b7e53c529ed80c03eaaeed1
+        timestamp: "2020-05-12T10:16:14Z"
+    RewardTransaction:
+      type: object
+      properties:
+        destination:
+          type: string
+        amount:
+          type: integer
+          format: int64
+    Height:
+      type: integer
+      format: int64
+    BlockReference:
+      type: object
+      properties:
+        hash:
+          type: string
+        height:
+          $ref: "#/components/schemas/Height"
+    Block:
+      type: object
+      properties:
+        hash:
+          type: string
+        height:
+          $ref: "#/components/schemas/Height"
+        transactions:
+          type: array
+          items:
+            type: string
+        parent:
+          type: array
+          items:
+            $ref: "#/components/schemas/BlockReference"
+        snapshot:
+          type: string
+        timestamp:
+          type: string
+          title: ISOString
+      example:
+        hash: 0923bd4a633f3040589547cfed00f2660fb736ce694199384f1df2adaac54fe6
+        height: 58184
+        transactions:
+          - adf1ba091a15a2dbj930a7245b36f8abbef7d9eaf0147466bc6069a3eacc4ef6
+          - 2fd12a358a59q6ee0f59687126ff7a984f993fa8002614169085348477ba5ba0
+        parent:
+          - hash: 2fd12a358a59q6ee0f59687126ff7a984f993fa8002614169085348477ba5ba2
+            height: 51232
+        snapshotHash: 1ae6e0f9161b7a29c96e8accb853fb479e0ea026832850ff5def773b5ff00c00
+        timestamp: "2020-05-12T10:16:14Z"
+    TransactionOriginal:
+      type: object
+    Transaction:
+      type: object
+      properties:
+        hash:
+          type: string
+        source:
+          type: string
+        destination:
+          type: string
+        amount:
+          type: integer
+          format: int64
+        fee:
+          type: integer
+          format: int64
+        parent:
+          type: object
+          properties:
+            hash:
+              type: string
+            ordinal:
+              type: integer
+              format: int64
+        snapshot:
```

Comment:
It will be changed to `snapshotHash` and also `snapshotOrdinal` will be returned